modules/standard/Time.chpl

```diff
@@ -161,9 +161,15 @@ module Time {
   /* End day-of-week enums */
 
   /* The minimum year allowed in `date` objects */
+  @deprecated("'MINYEAR' is deprecated; use `date.min.year` instead")
   param MINYEAR = 1;
   /* The maximum year allowed in `date` objects */
+  @deprecated("'MAXYEAR' is deprecated; use `date.max.year` instead")
   param MAXYEAR = 9999;
+  @chpldoc.nodoc
+  param _MINYEAR = 1;
+  @chpldoc.nodoc
+  param _MAXYEAR = 9999;
```

Comment:
I decided not to do that so that the `MINYEAR`/`MAXYEAR` constants look right in docs during the deprecation period. But also I don't feel strongly about it especially since they're deprecated and soon to be gone anyways. I can change it